### ISSUE TITLE
addition to overview
expressed in the germline: se...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#4924]
*Help Desk query collected when no chat operators were online. Follow up required.*

addition to overview
expressed in the germline: see
PMID: 25195067

**Reported by:** alex***** (alex********************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/http://www.wormbase.org/species/c_elegans/gene/WBGene00002136#0-g1c4-10">http://www.wormbase.org/species/c_elegans/gene/WBGene00002136#0-g1c4-10</a>
**Browser:** Safari 6.2.8

**Comment by draciti** [comment]
Hi Alexandra, thanks for reaching out.
I am including our concise description curator to see if she can add that info to the inx-14 Overview.

Best,

Daniela

**Comment by rankishore** [comment]
Replied to User:Hi Alexandra
Thanks for writing to WormBase.  The inx-14 description that currently appears in the Overview, is one of our older descriptions and we are in the process of weeding these out and replacing them with our automated descriptions, a project we recently developed in WormBase.  

The automated descriptions are based on data from current selected projects in WormBase like orthology, GO annotations, expression pattern curation etc, so are more current.  I’ve disabled the dumping of the old description, so the newer automated description for inc-14 can make it’s way to the website for the WS256 version of WormBase.  We just uploaded data for WS255, so apologies, we just missed this.  (The release schedule for WB is here: http://www.wormbase.org/about/release_schedule#01--10)
